### PR TITLE
fix(efb): Add custom attribution text

### LINKED ISSUE #5308: FlyPad shows "Leaflet"
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-07-10T11:13:17+00:00",
  "ref": "master",
  "sha": "a33bbcf89fd5a011163eee22d6c0eb4a8af46796",
  "actor": "davidwalschots",
  "event_name": "manual"
}
```

### Describe the bug

When I zoom in, a Leatflet page comes up. When I zoom out again, the normal FlyPad comes back but I can't click anything.

### Expected behavior

The bug is also in the new flyPad.

### Steps to reproduce

Click on "Openstreetmap" at the bottom right of the map on the dasboard

 Then comes the "Leaflet" page.

### References (optional)

https://user-images.githubusercontent.com/60877083/125431040-3b285173-3144-40d6-954f-248701bd35ab.mp4

### Additional info (optional)

_No response_

### Discord Username (optional)

matze[]# 2346

### PR DESCRIPTION
**Pull request by bouveng**
Fixes #5308 #3306

## Summary of Changes
Removes default attributions, adds a custom plain text one without links and moves this below the map.

Discord username: @bouveng

## Screenshots
attribution

## Testing instructions
Open EFB, observe new attribution text at the bottom right below the map.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/instruments/src/EFB/Assets/Efb.scss [hunk 1]
```diff
@@ -116,6 +116,20 @@ html body {
 
 .leaflet-container {
   @apply rounded-2xl;
+
+  .leaflet-control-attribution {
+    display: none;
```
Comment (hiaaryan) [inline src/instruments/src/EFB/Assets/Efb.scss:121]: `@apply hidden;`

File: src/instruments/src/EFB/Assets/Efb.scss [hunk 2]
```diff
@@ -116,6 +116,20 @@ html body {
...
+  }
+}
+
+.credits {
+  font-family: 'IBMPlexMono';
+  font-size: 10px;
+  letter-spacing: -0.5px;
+  color: rgba(255, 255, 255, .5);
+  display: block;
```
Comment (hiaaryan) [inline src/instruments/src/EFB/Assets/Efb.scss:130]: `@apply block;`

File: src/instruments/src/EFB/Assets/Efb.scss [hunk 3]
```diff
@@ -116,6 +116,20 @@ html body {
...
+  text-align: right;
```
Comment (hiaaryan) [inline src/instruments/src/EFB/Assets/Efb.scss:131]: `@apply text-right;`

File: src/instruments/src/EFB/Assets/Efb.scss [hunk 4]
```diff
@@ -116,6 +116,20 @@ html body {
...
+  margin-right: 5px;
```
Comment (hiaaryan) [inline src/instruments/src/EFB/Assets/Efb.scss:132]: please use `mr-1` or `mr-1.5`

File: src/instruments/src/EFB/Assets/Efb.scss [hunk 5]
```diff
@@ -116,6 +116,20 @@ html body {
...
   @apply rounded-2xl;
+
+  .leaflet-control-attribution {
+    display: none;
+  }
+}
+
+.credits {
+  font-family: 'IBMPlexMono';
+  font-size: 10px;
```
Comment (hiaaryan) [inline src/instruments/src/EFB/Assets/Efb.scss:127]: please use `text-xs` or `text-sm`

File: src/instruments/src/EFB/Assets/Efb.scss [hunk 6]
```diff
@@ -116,6 +116,20 @@ html body {
...
 .leaflet-container {
   @apply rounded-2xl;
+
+  .leaflet-control-attribution {
+    display: none;
+  }
+}
+
+.credits {
+  font-family: 'IBMPlexMono';
```
Comment (hiaaryan) [inline src/instruments/src/EFB/Assets/Efb.scss:126]: we can extend this to be used as default mono in tailwind config.

**Review by hiaaryan — CHANGES_REQUESTED** [review]

File: src/instruments/src/EFB/Assets/Efb.scss (hunk 6 above)
Comment (bouveng) [reply src/instruments/src/EFB/Assets/Efb.scss:126]: I think I already did it default mono in tailwind config, if I'm not mistaken. 😎

Comment (bouveng) [reply src/instruments/src/EFB/Assets/Efb.scss:126]: Yes. It was. `fontFamily: { mono: ['IBMPlexMono'], efb: ['NunitoSans'] },`

Comment (hiaaryan) [reply src/instruments/src/EFB/Assets/Efb.scss:126]: Cool, we can then use `@apply font-mono;` here.

**Comment by bouveng** [comment]
@hiaaryan Removed the `credits` class all together and went all in tailwind in markup instead, as it is supposed to be used.

**Review by hiaaryan — APPROVED** [review]

**Comment by ShadowFrost3** [comment]
Quality Assurance Trainee Report

Object of testing: #5400
Tier of Testing: 1
Date: 28/07/2021

Testing Process: Cold and dark a320, open tablet, view image of bottom left, appears to match PR
Negatives: N/A
Testing Results: Passed

image

**Pull request merged**